cmd/experimental/kueue-viz/backend/handlers/cluster_queues.go

```diff
@@ -0,0 +1,163 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+)
+
+// ClusterQueuesWebSocketHandler streams all cluster queues
+func ClusterQueuesWebSocketHandler(dynamicClient dynamic.Interface) gin.HandlerFunc {
+	clusterQueuesGVR := schema.GroupVersionResource{
+		Group:    "kueue.x-k8s.io",
+		Version:  "v1beta1",
+		Resource: "clusterqueues",
+	}
+
+	return GenericWebSocketHandler(dynamicClient, clusterQueuesGVR, "", func() (interface{}, error) {
```

Comment:
```suggestion
	return GenericWebSocketHandler(dynamicClient, ClusterQueueGVR(), "", func() (interface{}, error) {
```

Reply:
fixed, thanks
it was an incomplete refactoring